### PR TITLE
docs(api) document compilation object

### PR DESCRIPTION
**Pull request by EugeneHlushko**
Takes a bit off #3148 

Part of the methods dont have types and descriptions yet.

Reference: https://github.com/webpack/webpack/blob/master/lib/Compilation.js

### CODE REVIEW

File: src/content/api/compilation-object.md [hunk 1]
```diff
@@ -0,0 +1,322 @@
...
+`function`
+
+Returns Stats object for the current compilation.
+
+### addModule
+
+`function (module, cacheGroup)`
+
+Returns Stats object for the current compilation.
+
```
Comment (rishabh3112) [inline src/content/api/compilation-object.md]:
```suggestion

Parameters:

```

Comment (EugeneHlushko) [reply src/content/api/compilation-object.md]: ty! great catch

**Comment by EugeneHlushko** [comment]
@montogeek a reminder to take a look :) would to get this in as i have a page to add to API, which will mean a conflict in `sort` field

**Comment by montogeek** [comment]
Thanks!

**Pull request merged**